CONTRIBUTING.md

```
@@ -0,0 +1,173 @@
+4. You've **tested in the latest `twenty-*` theme**, and added deprecations for any
+   static-block `edit`/`save` changes.
+5. Bump the plugin header `Version:` and add a `CHANGELOG.md`/`readme.txt` changelog
+   entry when releasing changes — the release workflow reads the **PHP header version**.
+
+## Releases and auto-updates
+
+Releases are automatic on merge to `trunk` (see [How the monorepo works](#how-the-monorepo-works)).
+To make a plugin self-update on installed sites:
+
+1. Add the update URI to the plugin header so its hostname matches the update filter:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)